documentation/htc_workloads/workload_planning/roadmap.md

```diff
@@ -43,11 +43,11 @@ Learn more about the services provided by the OSG that can support your HTC work
 If you have not done so, apply for an account at <https://portal.osg-htc.org/application>. A Research Computing Facilitator will contact you within one business day to arrange a meeting to discuss your computational goals and to activate your account. 
 
 Users with accounts created starting in Spring 2023 have their accounts managed through a new account management service. To learn how to login to your account, visit: 
-- [Requesting an OSPool ap7.ospool.osg-htc.org Account](../../../overview/account_setup/ap7-access.md)
+- [Join and Log In to OSG Manage Access Points](../../../overview/account_setup/ap7-access.md)
```

Comment:
`OSG Manage Access Points` is a little bit difficult to parse. Is it a managed access point? Maybe quote it: `"OSG Manage" Access Points`